Zend/zend_language_parser.y

```diff
@@ -798,9 +798,9 @@ property_list:
 ;
 
 property:
-		T_VARIABLE { $$ = zend_ast_create(ZEND_AST_PROP_ELEM, $1, NULL); }
-	|	T_VARIABLE '=' expr
-			{ $$ = zend_ast_create(ZEND_AST_PROP_ELEM, $1, $3); }
+		T_VARIABLE backup_doc_comment { $$ = zend_ast_create(ZEND_AST_PROP_ELEM, $1, NULL, $2); }
+	|	T_VARIABLE '=' expr backup_doc_comment
+			{ $$ = zend_ast_create(ZEND_AST_PROP_ELEM, $1, $3, $4); }
```

Comment:
This should use `zend_ast_create_zval_from_str` instead of storing a raw string pointer.